libs/prisma-value/src/sql_ext.rs

```diff
@@ -36,7 +36,7 @@ impl<'a> From<PrismaValue> for Value<'a> {
             PrismaValue::Null => Value::Null,
             PrismaValue::Uuid(u) => u.to_string().into(),
             PrismaValue::List(l) => Value::Array(l.into_iter().map(|x| x.into()).collect()),
-            PrismaValue::Json(s) => Value::Text(s.into()),
+            PrismaValue::Json(s) => Value::Json(serde_json::from_str(&s).unwrap()),
```

Comment:
Should we unwrap there? It's a bit scary to me. Maybe we could delay this conversion to a fallible context with https://docs.rs/serde_json/1.0.53/serde_json/value/struct.RawValue.html

Reply:
nevermind, RawValue conversion is also fallible